Feat/speaker information

anyulled opened the pull request:
Draft concept for speakers activities section.

Missing confirmation from the sponsor.

icougil reviewed (commented):
Can't see the Sponsor logo, the style is not beautiful... 
(I understand that is a draft version...)
Where in the menu this will appear?

anyulled commented:
> Can't see the Sponsor logo, the style is not beautiful... (I understand that is a draft version...) Where in the menu this will appear?

I will add the logo when the company confirms the sponsorship and sends us their information. I still don't know where we could locate this, as it's a complete new section compared to previous editions.

anyulled commented:
> Can't see the Sponsor logo, the style is not beautiful... (I understand that is a draft version...) Where in the menu this will appear?

I added the sponsor logo, even though it's not confirmed yet. Also changed the style, including images and the latest information on the activities. This will have a link on the navigation menu.

icougil commented:
We should include Confluent in the list of sponsors as well, but for me looks ok 👍

anyulled commented:
I will include it in another pull request, as I just received their
sponsorship information.

Le ven. 23 juin 2023 à 15:44, Nacho Cougil ***@***.***> a
écrit :

> ***@***.**** approved this pull request.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dev-bcn/dev-bcn.github.io/pull/197#pullrequestreview-1495177532>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAYTBN3NQMNLDSG7SNA5OTXMWMSHANCNFSM6AAAAAAZCJSIIU>
> .
> You are receiving this because you were assigned.Message ID:
> ***@***.***>
>


-- 
*Anyul Rivas*
https://www.anyulled.tech
